fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake

```diff
@@ -42,20 +42,31 @@ find_package(fastrtps REQUIRED CONFIG)
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastcdr_MAJOR_MINOR_VERSION "${fastcdr_VERSION}")
 string(REGEX MATCH "^[0-9]+\\.[0-9]+" fastrtps_MAJOR_MINOR_VERSION "${fastrtps_VERSION}")
 
-find_library(FastRTPS_LIBRARY
+find_library(FastRTPS_LIBRARY_RELEASE
     NAMES fastrtps-${fastrtps_MAJOR_MINOR_VERSION} fastrtps)
 
-find_library(FastCDR_LIBRARY
+find_library(FastRTPS_LIBRARY_DEBUG
+    NAMES fastrtps${CMAKE_DEBUG_POSTFIX}-${fastrtps_MAJOR_MINOR_VERSION} fastrtps)
```

Comment:
What should be the correct way of exporting the libraries? On Linux and OSX, FastRTPS does not add any suffix if built with debug symbols.

Also, is the following correct? https://github.com/ros2/rmw_fastrtps/blob/7bbb69100ca2be642de4c7221b069e484c8c2160/fastrtps_cmake_module/cmake/Modules/FindFastRTPS.cmake#L67-L70 I added those variables to `find_package_handle_standard_args`, but I wonder what would happen if `rmw_fastrtps_cpp` is not built for debug.

In any case, given that the current nightly build is broken, if there's any way to solve this, feel free to make changes to this branch, the timezone difference just makes communication rather difficult :-/